hypha/settings/base.py

```diff
@@ -762,3 +762,12 @@
 
 MATOMO_URL = env.get('MATOMO_URL', False)
 MATOMO_SITEID = env.get('MATOMO_SITEID', False)
+
+# SAGE INTACCT Configuration
+if 'INITIALISE_INTACCT' in env:
+    INITIALISE_INTACCT = env['INITIALISE_INTACCT']
+    INTACCT_SENDER_ID = env['INTACCT_SENDER_ID']
+    INTACCT_SENDER_PASSWORD = env['INTACCT_SENDER_PASSWORD']
+    INTACCT_USER_ID = env['INTACCT_USER_ID']
+    INTACCT_COMPANY_ID = env['INTACCT_COMPANY_ID']
+    INTACCT_USER_PASSWORD = env['INTACCT_USER_PASSWORD']
```

Comment:
Need to set these variables to env for it to work.

Reply:
I prefer if we set bool True/False like we do in other places in base.py:

https://github.com/HyphaApp/hypha/blob/a5d174ce64950a33745144fcdfa61f01488d3417/hypha/settings/base.py#L731-L733

I also think we should rename INITIALISE_INTACCT to INTACCT_ENABLED. To follow the naming of current vars.

Reply:
Thanks @frjo, that is correct, have pushed the changes.